.gitmodules

```diff
@@ -76,3 +76,6 @@
 [submodule "auxil/prometheus-cpp"]
 	path = auxil/prometheus-cpp
 	url = https://github.com/zeek/prometheus-cpp
+[submodule "src/cluster/backend/zeromq/auxil/cppzmq"]
```

Comment:
>  I'm asking since it looks like it's available via packages on most systems

It is, but Ubuntu 20 ships with 4.7.0 and Centosstream 9 with 4.8.7. I've developed against 4.9.0, however. Both distros do provide a sufficient libzmq package.

This is mostly being nice to users so that they don't need to figure out how to install a recent cppzmq. cppzmq is also header-only, so this seemed fine.